documentation/requirements.txt

```diff
@@ -10,3 +10,4 @@ pyenchant
 sphinx-issues
 sphinx-argparse
 towncrier
+..
```

Comment:
This is probably harmful for users who already have installed cocotb with `-e`

Reply:
That will install cocotb into the virtualenv created for documentation, it won't collide.

Reply:
Not if I consume this file directly - you're assuming I came through the makefiles, which I did not.